.github/workflows/secp256r1_blake160.yml

```diff
@@ -0,0 +1,20 @@
+name: secp256r1_blake160
+
```

Comment:
This CI doesn't run. Try changing the file name to "rust.yml".

Reply:
It did run in [my repo](https://github.com/contrun/ckb-miscellaneous-scripts/runs/7355696867?check_suite_focus=true). This PR is not merged yet, so the github actions workflow actually does not exist in nervosnetwork's repo. I think aftermerging this, it will run in this repo.